third_party/amd/lib/TritonAMDGPUTransforms/BlockPingpong.cpp

```diff
@@ -137,6 +137,7 @@ void Pingponger::transformOnePPClusters(OpBuilder &builder, Location loc) {
 
   // Dot cluster #0
   updateOpInsertion(preDotBar);
+  dotOps[0]->emitRemark() << "Performed one ping pong cluster transformation\n";
```

Comment:
Nit: can we move these remarks to the end of the function right before the return? That way we keep the main logic together, and it's also very clear we are done entirely.

Similarly for other cases.